src/main/webapp/content/scss/global.scss

```
@@ -313,6 +313,8 @@ Custom alerts for notification
 .alerts {
     .alert {
         text-overflow: ellipsis;
+        display: flex;
+        justify-content: space-between;
```

```suggestion
        justify-content: space-between;

        .close {
            background: none;
            border: none;
        }
```
Additional rule to fix the button styling.